app/logging/logger.go

```diff
@@ -79,6 +81,5 @@ func ResetShared() {
 
 func new() *Logger {
 	logger := logrus.New()
-	log.SetOutput(logger.Writer())
```

Comment:
I am pretty sure there was a reason why it was there actually.

Ok, you do this, then the two  remaining lines should be merged.

Reply:
fixed